package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "records-server",
+  "version": "1.0.0",
+  "description": "An open source project to practice client-server intergration",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
It would be great to have a command to start the server, could you take care of it?

Reply:
done

Reply:
Thanks, good job.